earth_enterprise/src/common/packet.cpp

```diff
@@ -325,7 +325,7 @@ int etDataPacket::setFlatFileCount(int num)
   FILE *fp;
   char fname[500];
 
-  sprintf(fname, "%s.header", flatbasename);
+  snprintf(fname, 500, "%s.header", flatbasename);
```

Comment:
same as above. in this instance, the const would be used when fname is declared also

also, applies to the other instances in this file